.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy
+on: [push, pull_request]
+jobs:
+  deploy:
+    name: build dependencies & create artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
+      - name: Build JS dependencies
+        uses: PrestaShopCorp/github-action-build-js/12@dev
+        with:
+          cmd: yarn
+          path: ./_dev
+      - name: Install composer dependencies
+        run: composer install --no-dev -o
+      - name: Clean-up project
+        uses: PrestaShopCorp/github-action-clean-before-deploy@dev
```

Comment:
Before accepting your PR, can't we release github-action-clean-before-deploy to set a specific version there ?
Like `PrestaShopCorp/github-action-clean-before-deploy@v1` ?

Reply:
Is there really a use for doing it as a version and not using a branch?
Currently I use the dev branch because the github action is currently in code review for merge in master.

Reply:
i agree with @apacios, this is a dependency like any others, so if you update a branch you will possibly break every repository depending on it :/

Reply:
no problem when github actions will be on master I'll make a tag with a specific version

Reply:
The github actions are on the master branch and the tags have been created.